### DISCUSSION TITLE
Multi-level categories by media, add some free good PDF viewer

### DISCUSSION THREAD

**Discussion by gimail** [mediacms-io/mediacms#1097] [Feature Requests]
Good day, a big request to make three things optional, who needs includes: multi-level categories by media, add some free good PDF viewer, add the ability to enter a simple blog with pictures by categories too. If this is impossible, then most likely a methodological question, how best to do this? Thank you.

**Comment by saber13812002** [comment]
write a prompt for AI to develop this for you as another shell that call mediacms apis by /swagger that openapi provide it for you

**Comment by mgogoulos** [comment]
Looking back on this, out of the three requests here the one has been implemented (pdf support), plus a big part of the other (blog) is there as the ability to add custom pages and edit through wysiwyg editor is there.
